composer.json

```diff
@@ -16,7 +16,11 @@
   "require": {
     "adyen/php-api-library": "~2.1",
     "magento/framework": ">=101.0.8 <102 || >=102.0.1",
-    "magento/module-vault": "101.*"
+    "magento/module-vault": "101.*",
+    "php": "~7.1.3||~7.2.0||~7.3.0"
```

Comment:
As we support Magento 2.2.8 and higher the minimum php version we need to support is 7.0.13 https://github.com/magento/magento2/blob/2.2.8/composer.json#L11

Reply:
Just pushed a commit to match with Magento 2.2.8 :)